src/catalog/messages.js

```diff
@@ -0,0 +1,19 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+export default defineMessages({
+  'catalog.heading': {
```

Comment:
I personally try to use a simple id so I can use `messages.catalog_heading` or similar.